src/quicr_server.cpp

```diff
@@ -264,6 +280,37 @@ QuicRServer::TransportDelegate::on_connection_status(
   log_msg << "connection_status: cid: " << context_id
           << " status: " << int(status);
   server.log_handler.log(qtransport::LogLevel::debug, log_msg.str());
+
+  if (status == qtransport::TransportStatus::Disconnected) {
+    log_msg.str("");
```

Comment:
Seems odd, is this just a missed deletion?

Reply:
```log_msg.str(“”)``` reinitializes the output string stream. Might be more efficient with ```string()``` but in terms of performance, this is not a concern.